src/pages/Registration/Registration.tsx

```diff
@@ -0,0 +1,194 @@
+import { useNavigate } from 'react-router';
+import classes from './registration.module.css';
+import { FormEvent, ReactElement, useState } from 'react';
+import { sdk } from '../../services/sdk/create-client';
+import { BillingAdressOptions, LoginResponse } from '../../types/types';
+import AuthAlert from '../../components/common/auth-alert/AuthAlert';
+import SwitchButton from '../../components/common/switch-button/SwitchButton';
+import PopupForm from '../../components/common/popup-form/PopupForm';
+import validateForm from '../../utils/validate-form-register';
+import createUserData from '../../utils/create-user-data';
+import Tooltip from '../../components/common/tooltip/Tooltip';
+import FormInput from '../../components/common/input-form/FormInput';
+import CountrySelect from '../../components/common/input-form/CountrySelect';
+import ButtonSignUp from '../../components/common/form-links/ButtonSignUp';
+import SignInLink from '../../components/common/form-links/SignInLink';
+
+export default function Registration(): ReactElement {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [dob, setDob] = useState('');
+  const [streetName, setStreetName] = useState('');
+  const [city, setCity] = useState('');
+  const [postalCode, setPostalCode] = useState('');
+  const [country, setCountry] = useState('');
```

Comment:
While this solution is also fine, I would personally prefer to use 1 object instead of multiple useState.
An even better solution would be to use a library like formik.